implementations/c/ockam/channel/channel.h

```diff
@@ -0,0 +1,28 @@
+#ifndef OCKAM_CHANNEL_H
```

Comment:
Create rules for how header guards look like (I think there already exist such rules, though I've seen different approaches throughout the project headers)